system/modules/isotope/library/Isotope/Model/ProductCache.php

```diff
@@ -227,6 +228,10 @@ public static function deleteForPageAndModuleOrExpired($intPage, $intModule)
     public static function purge()
     {
         Database::getInstance()->query("TRUNCATE " . static::$strTable);
+
+        if (System::getContainer()->has('fos_http_cache.cache_manager')) {
+            System::getContainer()->get('fos_http_cache.cache_manager')->invalidateTags(['contao.db.tl_iso_product']);
```

Comment:
this would not clear the reader pages though, right?

Reply:
Only the product list uses the `ProductCache`.

Reply:
yes, but this method is called all over the place where the cache of products should be cleared. E.g. if the store configuration is changed, the reader pages might display a wrong currency. So it would kinda make sense to clear them as well? Although we would be better off actually implementing the correct tags. Which is why this might not make sense as a quick fix in 2.9 🙈

Reply:
> Although we would be better off actually implementing the correct tags.

I can add the additional cache tags in this PR if you want.

> Which is why this might not make sense as a quick fix in 2.9

Why? It's a bug in 2.9 and can easily be fixed there.

Reply:
idk, its also a feature of Contao that is not supported by Isotope 🤷

Reply:
Caching you mean?